content/posts/0.19-release.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Pyodide 0.19.0 release"
+date: 2022-01-07
+draft: true
+tags: ["internals"]
+author: "Hood Chatham and Roman Yurchak"
+showToc: true
+TocOpen: false
+draft: false
+hidemeta: false
+comments: false
+# description: ""
+disableHLJS: true # to disable highlightjs
+disableShare: false
+hideSummary: false
+searchHidden: true
+ShowReadingTime: true
+ShowBreadCrumbs: true
+ShowPostNavLinks: true
+---
+
+Pyodide 0.19 is a major release focused on improving the build system, updating
+core scientific dependencies, and improving the workflow for loading Python
+code.
```

Comment:
I moved most of the text that was here to a section below. Feel free to re-add some here if you want, but I think the introduction should be short and concrete, and some of that discussion goes a bit too much into details IMO.